### ISSUE TITLE
Bundle Filebeat with Elasticsearch

### ISSUE THREAD

**Issue by s-nel** [elastic/elasticsearch#50572]
We should bundle Filebeat with Elasticsearch. Logs are an important part of the observability story for ES, and the motivation for having Metricbeat bundled with Elasticsearch applies to Filebeat too. E.g. not having to install extra tools to observe ES.

Related to #49399

/cc @jasontedor @cachedout

**Comment by cachedout** [comment]
I've actually switched teams so I'm not longer involved in this issue. I will cc: @elastic/stack-monitoring so that they're aware, however.

**Comment by mark-vieira** [comment]
@jasontedor I vaguely remember some discussions around this and making it easier for Elasticsearch to ship logs to... Elasticsearch. I'm not sure we landed on how that might work. I'm not sure we came to any agreement on this, or if "embedding filebeat" is the prefered method. If we don't have plans on doing this I'd like to close this issue, or at least rephrase in terms of a general solution to the shipping logs problem.
